web/server/entrypoint.go

```diff
@@ -328,6 +330,14 @@ func (s *robotServer) serveWeb(ctx context.Context, cfg *config.Config) (err err
 				restartInterval = newRestartInterval
 
 				if mustRestart {
+					bufSize := 1 << 20
```

Comment:
I double checked the size of some of the hangs I have locally. I'm seeing ~250kb on the upper end so this looks good to me.